### PR TITLE
 osdc/Journaler: add 'stopping' check to various finish callbacks

### PR DESCRIPTION
**Pull request by ukernel**

### CODE REVIEW

**Comment by tchaikov** [comment]
retest this please

File: src/osdc/Journaler.h [hunk 1]
```diff
@@ -253,6 +253,7 @@ class Journaler {
   static const int STATE_ACTIVE = 3;
   static const int STATE_REREADHEAD = 4;
   static const int STATE_REPROBING = 5;
+  static const int STATE_STOPPING = -6;
```
Comment (batrick) [inline src/osdc/Journaler.h]: Why `-6`?

**Comment by batrick** [comment]
@ukernel this is good to merge after testing but I'm still waiting to hear back if the `-6` state should be `6`?

**Comment by ukernel** [comment]
because I wanted to differentiate it from other states.  It's 6 in updated commit

**Pull request merged**